### PR TITLE
Fix NGINX

### PR DESCRIPTION
**Pull request by ericgaspar**

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: conf/nginx.conf [hunk 1]
```diff
@@ -1,10 +1,13 @@
 #sub_path_only rewrite ^__PATH__$ __PATH__/ permanent;
 location __PATH__/ {
-    proxy_pass                  http://localhost:__PORT__/;
-    proxy_set_header            Host $host;
```
Comment (Josue-T) [inline conf/nginx.conf:4]: Just a question, why do you remove this ?

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Pull request closed without merging**